### ISSUE TITLE
portage-bashrc-mv deduplicates flags

### ISSUE THREAD

**Issue by Kawanaao** [vaeth/portage-bashrc-mv#40]
portage-bashrc-mv deduplicates flags when trying to use

```sh
-Xclang -mllvm -polly -Xclang -mllvm -other-flag
```

in LDFLAGS mllvm is deduplicated and only

```sh
-Xclang -mllvm -polly -Xclang -other-flag
```

remains, which is erroneous

**Comment by vaeth** [comment]
I am not sure whether it is a good idea to make arguments itself have arguments only because they match a certain string.
Are you sure that the double combination `-Xclang -mllvm` is really needed as a prefix to the actual argument? Why wouldn't `-mllvm` be sufficient?

**Comment by Kawanaao** [comment]
> I am not sure whether it is a good idea to make arguments itself have arguments only because they match a certain string. Are you sure that the double combination `-Xclang -mllvm` is really needed as a prefix to the actual argument? Why wouldn't `-mllvm` be sufficient?

Without adding -Xclang `-mllvm -arg` will also be passed to the linker, which cannot load compile-time-only plugins, otherwise it will cause warnings/errors as `clang++-18: warning: argument unused during compilation: '-mllvm -polly'`, in some cases this will lead to errors during linking

**Comment by vaeth** [comment]
Fixed in portage-bashrc-mv-22.3
